{"uuid": "87e9cd04-4086-4096-9adc-7831ba851ebc", "children": ["4b34afb5-1353-4a03-8d74-5eddccd9a779"], "befores": [{"name": "open_products_tab", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=112.0.5615.49)\nStacktrace:\n0   chromedriver                        0x0000000108856ae8 chromedriver + 4958952\n1   chromedriver                        0x000000010884e7b3 chromedriver + 4925363\n2   chromedriver                        0x00000001084101d6 chromedriver + 475606\n3   chromedriver                        0x00000001083e5cec chromedriver + 302316\n4   chromedriver                        0x000000010847ef2f chromedriver + 929583\n5   chromedriver                        0x0000000108494926 chromedriver + 1018150\n6   chromedriver                        0x00000001084799e3 chromedriver + 907747\n7   chromedriver                        0x0000000108443aef chromedriver + 686831\n8   chromedriver                        0x000000010844500e chromedriver + 692238\n9   chromedriver                        0x00000001088129ad chromedriver + 4680109\n10  chromedriver                        0x0000000108817c12 chromedriver + 4701202\n11  chromedriver                        0x000000010881e79f chromedriver + 4728735\n12  chromedriver                        0x0000000108818b7a chromedriver + 4705146\n13  chromedriver                        0x00000001087eb27e chromedriver + 4518526\n14  chromedriver                        0x0000000108838908 chromedriver + 4835592\n15  chromedriver                        0x0000000108838a87 chromedriver + 4835975\n16  chromedriver                        0x000000010884800f chromedriver + 4898831\n17  libsystem_pthread.dylib             0x00007fff2060b8fc _pthread_start + 224\n18  libsystem_pthread.dylib             0x00007fff20607443 thread_start + 15\n\n", "trace": "  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/conftest.py\", line 48, in open_products_tab\n    return open_sign_in_page.sign_in(get_user_data()[0], get_user_data()[1])\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/page_objs/sign_in_page.py\", line 42, in sign_in\n    self.fill_email_field(mail).fill_password_field(password).click_login_button()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/page_objs/sign_in_page.py\", line 27, in fill_email_field\n    self._send_keys(locator=self.__mail_input, value=user_mail)\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/utilities/web_ui/base_page.py\", line 20, in _send_keys\n    element = self._wait_until_element_located(locator)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/utilities/web_ui/base_page.py\", line 14, in _wait_until_element_located\n    return self.__my_waiter.until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 86, in until\n    value = method(self._driver)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/support/expected_conditions.py\", line 69, in _predicate\n    return driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 831, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "sign_in", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=112.0.5615.49)\nStacktrace:\n0   chromedriver                        0x0000000108856ae8 chromedriver + 4958952\n1   chromedriver                        0x000000010884e7b3 chromedriver + 4925363\n2   chromedriver                        0x00000001084101d6 chromedriver + 475606\n3   chromedriver                        0x00000001083e5cec chromedriver + 302316\n4   chromedriver                        0x000000010847ef2f chromedriver + 929583\n5   chromedriver                        0x0000000108494926 chromedriver + 1018150\n6   chromedriver                        0x00000001084799e3 chromedriver + 907747\n7   chromedriver                        0x0000000108443aef chromedriver + 686831\n8   chromedriver                        0x000000010844500e chromedriver + 692238\n9   chromedriver                        0x00000001088129ad chromedriver + 4680109\n10  chromedriver                        0x0000000108817c12 chromedriver + 4701202\n11  chromedriver                        0x000000010881e79f chromedriver + 4728735\n12  chromedriver                        0x0000000108818b7a chromedriver + 4705146\n13  chromedriver                        0x00000001087eb27e chromedriver + 4518526\n14  chromedriver                        0x0000000108838908 chromedriver + 4835592\n15  chromedriver                        0x0000000108838a87 chromedriver + 4835975\n16  chromedriver                        0x000000010884800f chromedriver + 4898831\n17  libsystem_pthread.dylib             0x00007fff2060b8fc _pthread_start + 224\n18  libsystem_pthread.dylib             0x00007fff20607443 thread_start + 15\n\n", "trace": "  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/page_objs/sign_in_page.py\", line 42, in sign_in\n    self.fill_email_field(mail).fill_password_field(password).click_login_button()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/page_objs/sign_in_page.py\", line 27, in fill_email_field\n    self._send_keys(locator=self.__mail_input, value=user_mail)\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/utilities/web_ui/base_page.py\", line 20, in _send_keys\n    element = self._wait_until_element_located(locator)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/utilities/web_ui/base_page.py\", line 14, in _wait_until_element_located\n    return self.__my_waiter.until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 86, in until\n    value = method(self._driver)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/support/expected_conditions.py\", line 69, in _predicate\n    return driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 831, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "fill_email_field", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=112.0.5615.49)\nStacktrace:\n0   chromedriver                        0x0000000108856ae8 chromedriver + 4958952\n1   chromedriver                        0x000000010884e7b3 chromedriver + 4925363\n2   chromedriver                        0x00000001084101d6 chromedriver + 475606\n3   chromedriver                        0x00000001083e5cec chromedriver + 302316\n4   chromedriver                        0x000000010847ef2f chromedriver + 929583\n5   chromedriver                        0x0000000108494926 chromedriver + 1018150\n6   chromedriver                        0x00000001084799e3 chromedriver + 907747\n7   chromedriver                        0x0000000108443aef chromedriver + 686831\n8   chromedriver                        0x000000010844500e chromedriver + 692238\n9   chromedriver                        0x00000001088129ad chromedriver + 4680109\n10  chromedriver                        0x0000000108817c12 chromedriver + 4701202\n11  chromedriver                        0x000000010881e79f chromedriver + 4728735\n12  chromedriver                        0x0000000108818b7a chromedriver + 4705146\n13  chromedriver                        0x00000001087eb27e chromedriver + 4518526\n14  chromedriver                        0x0000000108838908 chromedriver + 4835592\n15  chromedriver                        0x0000000108838a87 chromedriver + 4835975\n16  chromedriver                        0x000000010884800f chromedriver + 4898831\n17  libsystem_pthread.dylib             0x00007fff2060b8fc _pthread_start + 224\n18  libsystem_pthread.dylib             0x00007fff20607443 thread_start + 15\n\n", "trace": "  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/page_objs/sign_in_page.py\", line 27, in fill_email_field\n    self._send_keys(locator=self.__mail_input, value=user_mail)\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/utilities/web_ui/base_page.py\", line 20, in _send_keys\n    element = self._wait_until_element_located(locator)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/HW17/utilities/web_ui/base_page.py\", line 14, in _wait_until_element_located\n    return self.__my_waiter.until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 86, in until\n    value = method(self._driver)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/support/expected_conditions.py\", line 69, in _predicate\n    return driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 831, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/mikhaill/PycharmProjects/hillel_aqa_pytest/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "parameters": [{"name": "user_mail", "value": "'standard_user'"}], "start": 1681295372820, "stop": 1681295372825}], "parameters": [{"name": "mail", "value": "'standard_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1681295372820, "stop": 1681295372856}], "start": 1681295372818, "stop": 1681295372858}], "start": 1681295372818, "stop": 1681295373164}